### ISSUE TITLE
In FoldingPresets only Top Left, Top Wide, Left Wide are available.

### ISSUE THREAD

**Issue by Kovacs-3d** [ArshvirGoraya/Godot-Collapsible-Container#2]
Hello.
Can you help me why only Top Left, Top Wide, Left Wide are available in FoldingPresets?
I am using v4.3.stable.mono.official [77dcf97d8]
I tried everything and copied what you show in the tutorial video but it doesn't change.
![help](https://github.com/user-attachments/assets/19c925c8-ae8d-478e-b00d-9136314e4a40)

**Comment by ArshvirGoraya** [comment]
@Kovacs-3d 

If you click the `TIPS` button shown in the screenshot (bottom right button in the FoldingPresets) it will print this out in the console:
`
For more FoldingDirections: parent to a Container (a MarginContainer is recommended). Ensure that the Container has the same minimum size as this node for the intended effect. (Copy and paste the custom_minimum_size from this inspector into the MarginContainer's custom_minimum_size!)
`
In other words, just parent the collapsible container to any Container node, and make sure the parent Container node has the same `minimum size` as the collapsible container (when it is fully open).

The tutorial video is older than the FoldingDirections feature, so it has different steps to reproduce this effect.

**Comment by Kovacs-3d** [comment]
Ahh yes I missed that. Thank you!
